webpagesish/.gitignore

```diff
@@ -0,0 +1,28 @@
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+pnpm-debug.log*
+lerna-debug.log*
+
+node_modules
+.DS_Store
+dist
```

Comment:
We wouldn't gitignore `dist`, right?

Reply:
True

Reply:
> We wouldn't gitignore `dist`, right?

I was wondering the reason on this. A self-reloading build script would save space in this case. Is this a compromise to avoid build steps?

Reply:
Yep, this way the settings page works for people without installing node, `npm install`ing and running a build script. The bundle is pretty small anyways

Reply:
That's too bad. I wish we can have a full-fledged build script. At least this is one step of it.

Reply:
> At least this is one step of it.

Yeah, this would still be a step forwards. I don't think "saving space" is really a concern here.

Reply:
Also, there's nothing directly wrong with checking-in outputs. The Chrome DevTools team [even checks in node_modules](https://www.jackfranklin.co.uk/blog/check-in-your-node-dependencies/).